docs/getting-started/rulesets.md

```diff
@@ -64,6 +64,47 @@ rules:
 In most cases, you will want to operate on resolved document and therefore won't specify that property.
 You might find `resolved` useful if your rule requires access to $refs.
 
+### Given
```

Comment:
Nice, appreciate it!
Could we actually create a separate PR out of it so that it makes its way as a part of 5.0.1? Will be useful from the online docs perspective